### DISCUSSION TITLE
Backup and import

### DISCUSSION THREAD

**Discussion by highfillgoods** [usememos/memos#3376] [Ideas]
I'm racking my brain on trying to tar ball clone a container, then import that as docker container on another server. Would be cool if in the site we can have a backup option to import or export data rather than use docker.

**Comment by DrSpaldo** [comment]
Yep, I just came to the github to work out how I can move my memos from Unraid to Proxmox.

**Comment by DrSpaldo** [comment]
Any update on this? How we can backup the old instance and then import into the new one?

**Reply by highfillgoods** [reply]
If you are on a vm and in Linux, You can just copy from in the home folder the memos database folder. It's the hidden folder .memos. Copy that from one vm to another. this is a full backup I've noticed.    If you going from unraid to proxmox I would do scp - r user@ipofunraid:/home/billy/.memos /home/localuserbobby/Downloads..... Then do abother scp to place this memos backup in proxmox

**Reply by DrSpaldo** [reply]
> If you are on a vm and in Linux, You can just copy from in the home folder the memos database folder. It's the hidden folder .memos. Copy that from one vm to another. this is a full backup I've noticed. If you going from unraid to proxmox I would do scp - r user@ipofunraid:/home/billy/.memos /home/localuserbobby/Downloads..... Then do abother scp to place this memos backup in proxmox

Thanks for the reply @highfillgoods . I’ve swapped to Proxmox and both are LXC’s, so I ended up using various docker commands to download and transfer but got it working.

good reply for others searching for this issue. Would be much easier if there were a backup & restore option inside the app but good this thread is here for now

**Comment by vibbe** [comment]
+1 for this, should be prioritized!

**Reply by vibbe** [reply]
This is already a discussion about this but rather in Chinese, just so we can link them together, English - Chinese (whatever you prefer) 数据怎么备份保存？
#154

**Reply by vibbe** [reply]
This is the original discussion Ability to Fully Export and Import All Memos and Instance Configurations [https://github.com/usememos/memos/issues/2000](url) was closed as not planned, unfortunate.
